docs/deployment/add-ons/storage/efs/auto-efs-setup.py

```diff
@@ -540,50 +519,58 @@ def footer():
 
 parser = argparse.ArgumentParser()
 parser.add_argument(
-    '--region',
+    "--region",
     type=str,
     metavar="CLUSTER_REGION",
-    help='Your cluster region code (eg: us-east-2)',
-    required=True
+    help="Your cluster region code (eg: us-east-2)",
+    required=True,
 )
 parser.add_argument(
-    '--cluster',
+    "--cluster",
     type=str,
     metavar="CLUSTER_NAME",
-    help='Your cluster name (eg: mycluster-1)',
-    required=True
+    help="Your cluster name (eg: mycluster-1)",
+    required=True,
 )
-EFS_FILE_SYSTEM_NAME_DEFAULT = "KubeflowEfs"
+EFS_FILE_SYSTEM_NAME_DEFAULT = "Kubeflow-efs"
 parser.add_argument(
-    '--efs_file_system_name',
+    "--efs_file_system_name",
     type=str,
     default=EFS_FILE_SYSTEM_NAME_DEFAULT,
     help=f"Default is set to {EFS_FILE_SYSTEM_NAME_DEFAULT}",
-    required=False
+    required=False,
 )
 EFS_SECURITY_GROUP_NAME_DEFAULT = "KubeflowEfsSecurityGroup"
 parser.add_argument(
-    '--efs_security_group_name',
+    "--efs_security_group_name",
     type=str,
     default=EFS_SECURITY_GROUP_NAME_DEFAULT,
     help=f"Default is set to {EFS_SECURITY_GROUP_NAME_DEFAULT}",
-    required=False
+    required=False,
 )
 EFS_PERFORMANCE_MODE_DEFAULT = "generalPurpose"
 parser.add_argument(
-    '--efs_performance_mode',
+    "--efs_performance_mode",
     type=str,
     default=EFS_PERFORMANCE_MODE_DEFAULT,
     help=f"Default is set to {EFS_PERFORMANCE_MODE_DEFAULT}",
-    required=False
+    required=False,
 )
 EFS_THROUGHPUT_MODE_DEFAULT = "bursting"
 parser.add_argument(
-    '--efs_throughput_mode',
+    "--efs_throughput_mode",
     type=str,
     default=EFS_THROUGHPUT_MODE_DEFAULT,
     help=f"Default is set to {EFS_THROUGHPUT_MODE_DEFAULT}",
-    required=False
+    required=False,
+)
+DEFAULT_DIRECTORY_PATH = ""
+parser.add_argument(
+    "--directory",
+    type=str,
+    default=DEFAULT_DIRECTORY_PATH,
+    help=f"Specify the path to the source files if different. Default is set to empty.",
```

Comment:
Does this directory contain EFS specific source files or what more is it intended to contain?

Reply:
Yes it contains only EFS specific files. This parameter should ideally not be required by users. We need this to be able to specify the path to the sc.yaml file from the tests.